proposals/low-level-struct-improvements.md

```diff
@@ -152,12 +152,35 @@ A `ref` field can be combined with `readonly` modifiers in the following ways:
 - `ref readonly`: this is a field that can be ref re-assigned but cannot be value assigned at any point. This how an `in` parameter could be ref re-assigned to a `ref` field.
 - `readonly ref readonly`: a combination of `ref readonly` and `readonly ref`. 
 
+```
+ref struct ReadOnlyExample
+{
+    ref readonly int Field1;
+    readonly ref int Field2;
+    readonly ref readonly int Field3;
+
+    void Uses(int[] array)
+    {
+        Field1 = ref array[0];  // Okay
+        Field1 = array[0];      // Error: can't assign ref readonly value (value is readonly)
+        Field2 = ref array[0];  // Error: can't repoint readonly ref
+        Field2 = array[0];      // Okay
+        Field3 = ref array[0];  // Error: can't assign ref readonly value (value is readonly)
+        Field3 = array[0];      // Error: can't repoint readonly ref/ Error
+    }
```

Comment:
Should the error messages for the two `Field3` examples be reversed?

Reply:
good catch.